scm/driver/github/webhook.go

```diff
@@ -352,6 +439,95 @@ type (
 		Repository repository `json:"repository"`
 		Sender     user       `json:"sender"`
 	}
+
+	pipelineHook struct {
+		Action      string `json:"action"`
+		WorkflowRun struct {
+			ID               int64   `json:"id"`
+			Name             string  `json:"name"`
+			NodeID           string  `json:"node_id"`
+			HeadBranch       string  `json:"head_branch"`
+			HeadSHA          string  `json:"head_sha"`
+			Path             string  `json:"path"`
+			DisplayTitle     string  `json:"display_title"`
+			RunNumber        int     `json:"run_number"`
+			Event            string  `json:"event"`
+			Status           string  `json:"status"`
+			Conclusion       *string `json:"conclusion"`
+			WorkflowID       int64   `json:"workflow_id"`
+			CheckSuiteID     int64   `json:"check_suite_id"`
+			CheckSuiteNodeID string  `json:"check_suite_node_id"`
+			URL              string  `json:"url"`
+			HtmlURL          string  `json:"html_url"`
+			PullRequests     []struct {
+				URL    string `json:"url"`
+				ID     int64  `json:"id"`
+				Number int    `json:"number"`
+				Head   gitRef `json:"head"`
+				Base   gitRef `json:"base"`
+			} `json:"pull_requests"`
+			CreatedAt          string    `json:"created_at"`
+			UpdatedAt          time.Time `json:"updated_at"`
+			Actor              owner     `json:"actor"`
+			RunAttempt         int       `json:"run_attempt"`
+			RunStartedAt       time.Time `json:"run_started_at"`
+			TriggeringActor    owner     `json:"triggering_actor"`
+			JobsURL            string    `json:"jobs_url"`
+			LogsURL            string    `json:"logs_url"`
+			CheckSuiteURL      string    `json:"check_suite_url"`
+			ArtifactsURL       string    `json:"artifacts_url"`
+			CancelURL          string    `json:"cancel_url"`
+			RerunURL           string    `json:"rerun_url"`
+			PreviousAttemptURL *string   `json:"previous_attempt_url"`
```

Comment:
remove pointer, use `null.String` instead (see examples in other files)

Reply:
Done